build-data/static/surrogates.json

```diff
@@ -13,10 +14,12 @@
         ["googletagservices.com/tag/js/gpt.js", "gpt.js"]
     ],        
     "googlesyndication.com": [
-        ["googlesyndication.com/adsbygoogle.js", "adsbygoogle.js"]
+        ["googlesyndication.com/adsbygoogle.js", "adsbygoogle.js"],
+        ["googlesyndication.com/pagead/js/adsbygoogle.js", "adsbygoogle.js"]
     ],
     "doubleclick.net": [
-        ["doubleclick.net/instream/ad_status.js", "ad_status.js"]
+        ["doubleclick.net/instream/ad_status.js", "ad_status.js"],
+        ["doubleclick.net/tag/js/gpt.js", "gpt.js"]
```

Comment:
just to be sure - we are ignoring subdomains here right? https://securepubads.g.doubleclick.net/tag/js/gpt.js would get redirected by this rule?